src/ui/public/vis/agg_config.js

```diff
@@ -176,9 +177,9 @@ class AggConfig {
    * @return {void|Object} - if the config has a dsl representation, it is
    *                         returned, else undefined is returned
    */
-  toDsl() {
+  toDsl(aggs) {
```

Comment:
Could you please also update the doc block here? on the doc the param is called `@param  {AggConfig} aggConfig `. Or maybe change the `aggs` param to `aggConfig` ti keep it a bit more readable